Automattic/sensei
# Use bulk_edit_posts hook for bulk edit handler

This PR addresses the issue with performance degradation, noticed here: https://github.com/Automattic/sensei/pull/7523#pullrequestreview-1916445025.

## Proposed Changes

* Use bulk_edit_posts hook for bulk edit handler.

## Testing Instructions

1. Create a few courses with lessons.
2. Go to lessons and use bulk edit to update lesson course and complexity, for example.
3. Make sure the changes were applied to selected lessons.
4. Make sure Quick Edit works as well.

## Pre-Merge Checklist
